uber/models/attendee.py

```diff
@@ -589,16 +589,17 @@ def calculate_badge_cost(self, use_promo_code=True):
         registered = self.registered_local if self.registered else None
         base_badge_price = self.base_badge_price or c.get_attendee_price(registered)
 
+        age_discountable_badge_types = [c.ATTENDEE_BADGE]
+
         if self.paid == c.NEED_NOT_PAY:
             return 0
         elif self.overridden_price is not None:
             return self.overridden_price
         elif self.is_dealer:
             return c.DEALER_BADGE_PRICE
-        elif self.badge_type == c.ATTENDEE_BADGE and self.age_discount != 0:
+        elif self.badge_type in age_discountable_badge_types and self.age_discount != 0:
```

Comment:
If we move `age_discountable_badge_types` to the Attendee class, this will have to be `self.age_discountable_badge_types`